packages/i18n/src/test/index.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable @wordpress/i18n-text-domain */
+/* eslint-disable @wordpress/i18n-translator-comments */
```

Comment:
Nit: Can be:

```suggestion
/* eslint-disable @wordpress/i18n-text-domain, @wordpress/i18n-translator-comments */
```

Depending if it was intentional to split them across multiple lines for line-length's sake.